Tests/js/views/fields/assets/ez-media-view-tests.js

```diff
@@ -0,0 +1,158 @@
+                        mimeType: 'video/mp4',
+                        path: '',
+                        uri: '/var/marcus.avi',
+                        width: '42',
+                    }
+                };
+                this.isEmpty = false;
+                this.view = new Y.eZ.MediaView({
+                    fieldDefinition: this.fieldDefinition,
+                    field: this.field
+                });
+
+                // phantomjs does not support the video tag and the associated
+                // events so to simulate error events, we let
+                // YUI consider those events as custom events, this way,
+                // Y.Node.fire() can be used.
+                this.origErrorCfg = Y.Node.DOM_EVENTS.error;
+                delete Y.Node.DOM_EVENTS.error;
+            },
+
+            "Test field value": function () {
+                this._testValue(
+                    this.field.fieldValue, this.field.fieldValue, "The value in the template should be the fieldValue"
+                );
+            },
+
+            "Test empty field value": function () {
+                this._testValue(
+                    null, null, "The value in the template should be null"
+                );
+            },
+
+            "Test isEmpty with an empty field value": function () {
+                this._testIsEmpty(
+                    {fieldValue: null}, true,
+                    "null should be considered empty"
+                );
+            },
+
+            "Test isEmpty with a field value": function () {
+                this._testIsEmpty(
+                    this.field, false,
+                    "Value should not be considered empty"
+                );
+            },
+
+            "Test isAudio parameter is true": function () {
+                var origTpl = this.view.template;
+
+                this.view.set('field', {fieldValue: this.fieldValue});
+                this.view.template = function (variables) {
+                    Assert.isTrue(variables.isAudio, "isAudio should be true");
+                    return origTpl.apply(this, arguments);
+                };
+                this.view.render();
+            },
+
+            "Test isAudio parameter is false": function () {
+                var origTpl = this.view.template,
+                    fieldDefiniton = this.fieldDefinition;
+
+                fieldDefiniton.fieldSettings.mediaType = "TYPE_HTML5_VIDEO";
+
+                this.view.set('field', {fieldValue: this.fieldValue});
+                this.view.template = function (variables) {
+                    Assert.isFalse(variables.isAudio, "isAudio should be false");
+                    return origTpl.apply(this, arguments);
+                };
+                this.view.render();
+            },
+
+            tearDown: function () {
+                this.view.destroy();
+            },
+
+            "Should track the error event (audio)": function () {
```

Comment:
why only for audio ? Actually, there's nothing specific to audio or video in the error tracking, so one test for both is enough.